kbfsfuse/main.go

```diff
@@ -24,6 +24,7 @@ var runtimeDir = flag.String("runtime-dir", os.Getenv("KEYBASE_RUNTIME_DIR"), "r
 var label = flag.String("label", os.Getenv("KEYBASE_LABEL"), "label to help identify if running as a service")
 var mountType = flag.String("mount-type", defaultMountType, "mount type: default, force, none")
 var version = flag.Bool("version", false, "Print version")
+var servicemount = flag.Bool("service-mount", false, "get mount path from serfice")
```

Comment:
Let's just keep this feature windows-only for now, and we can add it to linux/OSX if we need it.

@strib the feature is that we need to enable the changing of the "drive letter" from the Electron interface.  So therefore need a way to update the configuration and then relaunch kbfsfuse with the new parameter.  The complication is that a process called the "watchdog" launches kbfsfuse on windows and can't conveniently access keybase configuration.  So this technique seemed simplest --- to ask for the service where to mount on startupt.